test/specs/EventsSpec.js

```diff
@@ -21,325 +21,355 @@ describe("Event Tests", function() {
     });
 
     describe("Mouse Events", function() {
+      var spy;
 
-      it("'slideStart' event is triggered properly and can be binded to", function() {
-        testSlider.on('slideStart', function() {
-          flag = true;
-        });
-        testSlider.data('slider')._mousedown(mouse);
-        expect(flag).toBeTruthy();
+      beforeEach(function() {
+        spy = jasmine.createSpy('spy');
       });
 
-      it("'slide' event is triggered properly and can be binded to", function() {
-        testSlider.on('slide', function() {
-          flag = true;
+      it("'slideStart' event is triggered properly and can be binded to", function(done) {
+        testSlider.on('slideStart', spy);
+        testSlider.data('slider')._mousedown(mouse);
```

Comment:
The following code is not good practice. The unit tests should avoid making calls to private methods such as `_mousedown()` or `_mouseup` etc. The proper solution is to use a `dispatchEvent()` call instead. However, to save you the work, this can be fixed in another pull request because this pattern occurs throughout this test file and others.
```js
testSlider.data('slider')._mousedown(mouse);
```

Reply:
I'll address this in another pull request (when I get time).